pkgs/desktops/gnome-3/extensions/text-translator/fix-deps.patch

```diff
@@ -0,0 +1,58 @@
+diff --git a/extension.js b/extension.js
+index 421a2aa..6b84ef9 100644
+--- a/extension.js
++++ b/extension.js
+@@ -6,6 +6,7 @@
+             GIRepository.Repository.prepend_search_path("/usr/lib/" + path);
+         }
+     );
++    GIRepository.Repository.prepend_search_path('@clutter@');
+ })();
+ 
+ const St = imports.gi.St;
+diff --git a/google_tts.js b/google_tts.js
+index 469e29d..0daaa1b 100644
+--- a/google_tts.js
++++ b/google_tts.js
+@@ -8,6 +8,8 @@ var GoogleTTS = class GoogleTTS {
+     constructor() {
+         Gst.init(null);
+ 
++        imports.gi.Gst.Registry.get().scan_path('@gstPlugins@');
```

Comment:
Gst is already imported above. And also can path only supports a single path so having plural there can be misleading:

```suggestion
+        Gst.Registry.get().scan_path('@gstBase@');
```

Reply:
Got it, done!